sdk/keyvault/azure-security-keyvault-certificates/pom.xml

```diff
@@ -97,7 +97,14 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-test</artifactId>
-      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
+      <version>1.2.0-beta.1</version> <!-- {x-version-update;unrelease_com.azure:azure-core-test;current} -->
```

Comment:
This tag is wrong, unreleased dependencies should only ever be dependency. The tag should be x-version-update;unreleased_com.azure:azure-core-test;dependency. As it is right now it'll definitely fail the analyze step's version scanner.